general/administration/installing.md

```diff
@@ -27,25 +32,59 @@ The Jellyfin Docker image is available on [Docker Hub](https://hub.docker.com/r/
     `--volume /path/to/cache:/cache \`  
     `--volume /path/to/media:/media \`  
     `--net=host \`  
-    `jellyfin/jellyfin`  
+    `jellyfin/jellyfin:latest`  
   
 Alternatively, using docker-compose:  
 ```
 version: "3"  
 services:  
-    jellyfin:  
-      image: jellyfin/jellyfin  
-      network_mode: "host"  
-      volumes:  
-        - /path/to/config:/config  
-        - /path/to/cache:/cache  
-        - /path/to/media:/media  
+  jellyfin:  
+    image: jellyfin/jellyfin:latest
+    container_name: jellyfin
+    network_mode: "host"  
+    volumes:  
+      - /path/to/config:/config  
+      - /path/to/cache:/cache  
+      - /path/to/media:/media  
+    environment:
+      APP_UID: 1000
+      APP_UID: 1000
+      TZ: Europe/Paris
+      UMASK_SET: 022
+      GIDLIST=100 #A comma-separated list of additional GIDs to run emby as (default 2)#
```

Comment:
There is no such environment variable in the Jellyfin Docker image. If you need this capability, use Docker Compose spec v2.4 and the `group_add` feature. https://docs.docker.com/compose/compose-file/compose-file-v2/#group_add

Spec v3 removed it.

Reply:
what variables are supported? I tried running APP_UID since that's what the documentation had and it didn't work.

Reply:
There are no environment variables to modify the user and groups. The expectation is to use Docker-native features for that.

I'm not sure what env vars are supported, but I don't believe the UMASK_SET nor TZ are.

Reply:
Would you be opposed to PR's related to adding ENVs to the official dockerfile?